course/page/code_feedback.py

```diff
@@ -135,16 +135,13 @@ def check_list(self, name, ref, data, entry_type=None):
     def check_scalar(self, name, ref, data, accuracy_critical=True,
             rtol=1e-5, atol=1e-8, report_success=True, report_failure=True):
         import numpy as np
+        import sympy as sp
```

Comment:
I'd rather not. `import sympy` looks innocuous, but it takes about a second--so we implicitly add that overhead to every autograder submission, whether it uses sympy or not.